src/systems/frequency_system.C

```diff
@@ -112,14 +112,18 @@ void FrequencySystem::init_data ()
   // make sure we have frequencies to solve for
   if (!_finished_set_frequencies)
     {
+      // not supported for now
+      if (parameters.have_parameter<unsigned int> ("n_frequencies"))
+        libmesh_error_msg("ERROR: Not supported");
+
       // when this system was read from file, check
       // if this has a "n_frequencies" parameter,
       // and initialize us with these.
       if (es.parameters.have_parameter<unsigned int> ("n_frequencies"))
         {
 #ifndef NDEBUG
           const unsigned int n_freq =
-            es.parameters.get<unsigned int>("n_frequencies");
+            es.para) :<unsigned int>("n_frequencies");
```

Comment:
Looks like a search-and-replace went badly awry somewhere.

Reply:
either that or switching tabs then starting typing in the console but it's still typing on the text editor

Reply:
error was in two places so def search and replace